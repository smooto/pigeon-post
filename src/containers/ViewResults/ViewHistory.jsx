import React from 'react';

const ViewResults = () => (
  <section>
    <h2>Results</h2>
    <div>
      {/* <pre>
        "example": {
          "json" : "here's some example JSON"
        }
      </pre> */}
    </div>
  </section>
);

export default ViewResults;
