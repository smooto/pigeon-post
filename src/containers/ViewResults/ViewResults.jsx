import React from 'react';
import JSONPretty from 'react-json-pretty';

const ViewResults = ({ data }) => (
  <section>
    <h2>Results</h2>
    <JSONPretty data={data}></JSONPretty>
  </section>
);

export default ViewResults;
