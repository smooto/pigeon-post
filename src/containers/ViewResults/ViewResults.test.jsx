import React from 'react';
import { shallow } from 'enzyme';
import ViewResults from './ViewResults';

describe('ViewResults component', () => {
  it('renders ViewResults', () => {
    const wrapper = shallow(<ViewResults data={[{}]} />);
    expect(wrapper).toMatchSnapshot();
  });
});
