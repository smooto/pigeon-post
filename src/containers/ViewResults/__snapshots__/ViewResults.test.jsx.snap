// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewResults component renders ViewResults 1`] = `
<section>
  <h2>
    Results
  </h2>
  <JSONPretty
    data={
      Array [
        Object {},
      ]
    }
    json=""
    silent={true}
    space={2}
    themeClassName="__json-pretty__"
  />
</section>
`;
